kivymd/theming.py

```diff
@@ -711,6 +711,43 @@ def _get_op_divider_color(self) -> list:
     property is readonly.
     """
 
+    def _get_disabled_primary_color(self, opposite: bool = False) -> list:
+        theme_style = self._get_theme_style(opposite)
+        lum = sum(self.primary_color[0:3]) / 3.0
+        if theme_style == "Light":
+            a = 0.38
+        elif theme_style == "Dark":
+            a = 0.50
+        return [lum, lum, lum, a]
+
+    disabled_primary_color = AliasProperty(
+        _get_disabled_primary_color, bind=["theme_style"]
+    )
+    """
+    The greyscale disabled version of the current application theme color
+    in ``rgba`` format.
+
+    :attr:`disabled_primary_color`
+    is an :class:`~kivy.properties.AliasProperty` that returns the value
+    in ``rgba`` format for :attr:`disabled_primary_color`,
+    property is readonly.
```

Comment:
If you are adding a new property, include it in the docstring - https://github.com/kivymd/KivyMD/blob/master/kivymd/uix/textfield/textfield.py#L605

Reply:
`    ... versionadded:: 1.0.0`

added for new properties (line_color_disabled and theme_icon_style).

Reply:
You have not added an `versionadded` instruction

Reply:
sorry, you beat me to pushing the changes by about 3 minutes...